packages/venia-concept/src/ServiceWorker/Utilities/__tests__/imageCacheHandler.spec.js

```diff
@@ -11,6 +13,24 @@ import {
 import { __handlers__ } from '../messageHandler';
 import { isFastNetwork } from '../networkUtils';
 
+jest.mock('workbox-cacheable-response', () => {
+    return {
+        CacheableResponsePlugin: jest.fn().mockImplementation()
+    };
+});
+
+jest.mock('workbox-expiration', () => {
+    return {
+        ExpirationPlugin: jest.fn().mockImplementation()
+    };
+});
+
+jest.mock('workbox-strategies', () => {
+    return {
+        CacheFirst: jest.fn().mockImplementation()
```

Comment:
Is `jest.fn().mockImplementation()` the same as `jest.fn()`?

Reply:
Oh, yes I think so. https://jestjs.io/docs/en/mock-function-api#mockfnmockimplementationfn

> Note: jest.fn(implementation) is a shorthand for jest.fn().mockImplementation(implementation).